instrumentation/active_job/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History: opentelemetry-instrumentation-active_job
 
+### v0.7.0 / 2023-11-22
+
+* BREAKING CHANGE: Drop Rails 6.0 EOL
+
+* ADDED: Drop Rails 6.0 EOL
+
```

Comment:
```suggestion

* BREAKING CHANGE: Use ActiveSupport Instrumentation instead of Monkey Patches

* CHANGED: Use ActiveSupport Instrumentation instead of Money Patches [#677](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/677)
```